### PR TITLE
Contact sec

### PR DESCRIPTION
**Pull request by misheck12**
Today I update my mobile portfolio with the following;

> Add partners to collaborate on this project 

> Create a form using HTML and CSS

> Contribute to this project by adding CSS

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -349,3 +351,69 @@ a .btn-two {
...
+.contact-form {
+  margin-top: 90px;
+  flex-direction: column;
+  justify-content: start;
+  background-image: url(images/contact-b.png);
+  background-size: contain;
+  border: 1px solid white;
+  border-top-left-radius: 110px;
+  text-align: center;
+}
```
Comment (CollinsTatang) [inline style.css:367]: - [x] Kindly work on your style again to fit the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Your design:
formstyle
Figma design, please follow it closely 
figma

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @misheck12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Gitflow flow was correctly used ✔️ 

## Required Changes ♻️

image
 

- [ ] Please kindly check the inline comment under the pull request.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (dfizzkruah93) [reply style.css:367]: I haven't done this project yet. Why am I receiving required changes for project I haven't create on my Dashboard ?

Comment (CollinsTatang) [reply style.css:367]: Sorry @dfizzkruah93  I mistaken mention your name in the review

Comment (misheck12) [reply style.css:367]: Hello @CollinsTatang thank you for the feedback it has pushed me to researching and learn more. However, i have noticed that the form is stable in some Mobile views and others not especially those that go beyond the recommended width. 
Am writing to you to share your opinion over this problem. Either through helping to it or anything that will work for you.

thank you being a hero . you can reach on slack as misheck livingif or faster replies.

happy coding

Comment (prolajumokeoni) [reply style.css:367]: Hi @misheck12, were you able to contact the previous reviewer

Comment (misheck12) [reply style.css:367]: Hey @prolajumokeoni , I have contacted him on slack but he is not replying

Comment (prolajumokeoni) [reply style.css:367]: - [x] Currently what is left is the horizontal scroll, I suggest you go through your CSS and check widths, margins, and padding they could also add a horizontal scroll, locally everything is fine on my end but when deployed I see the horizontal scroll  😉

File: index.html [hunk 2]
```diff
@@ -176,6 +176,21 @@ <h3 class="lang-font">Skills</h3>
...
+          you
+          need built or a project that needs coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/mrgjqpvb" method="POST" class="con-form">
+          <input type="text" name="_name" value="Yeremias" maxlength="30" class="form-text" placeholder="name" required>
+          <input type="email" name="_reply" value="natajayanj14@gmail.com" class="form-text" placeholder="email" required>
+          <textarea name="_message" placeholder="Write your message here" maxlength="500"
+              class="form-text message" required></textarea>
+          <button class="con-but" type="submit">Get in touch</button>
+      </form>
+    </div>
```
Comment (prolajumokeoni) [inline index.html:192]: ## OPTIONAL 

abc

- [x] Kindly add the background icons check screen shot above

**Review by prolajumokeoni — APPROVED** [review]
Hi @misheck12 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


153888587-3e47b8be-12ea-47db-a3c2-33122e091533

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (misheck12) [reply index.html:192]: Hey @prolajumokeoni thank you so much. will work on the icon background .

thank you once more

**Pull request merged**